iothub/device/src/TransportSettings/QualityOfService.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices.Client
+{
+    /// <summary>
+    /// The policy for which a particular message will be sent. Note that IoT hub does not support QoS 2.
+    /// </summary>
+    public enum QualityOfService
```

Comment:
We expose this to users?

Reply:
Users can choose the QoS of the MQTT messages they send, yes